src/main/java/core/basesyntax/service/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String messege) {
+        super(messege);
```

Comment:
You need to move this class to a new package and add a new class Exception with a NULL Exception.